Formula/voltdb.rb

```diff
@@ -14,11 +14,14 @@ class Voltdb < Formula
   end
 
   depends_on :ant => :build
+  depends_on "cmake" => :build
+
+  patch :DATA
```

Comment:
We could use an inreplace temporarily but long-term we need to get this working without any patches.